File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {    
        height = get_int("Height? \n"); //asks for the height of the leaders
    }
    while (height < 1 || height > 8);  //repeat if height is not between 1 and 8

    for(int i = 0; i < height ; i++)  //loop for lines
    {
        for(int j = height-1; j > -1 ; j--)  // loop for columns left side
        {
            if (i < j)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        {
            printf("  ");  //space between each side
        }
        for(int j = 0; j < height ; j++)   // loops for columns right side
        {
            if (i >= j)
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: mario.py
from cs50 import get_int

i, j, k, height = 0, 0, 0, 0 #declare variables

while True:
    height = get_int("Height: ") #asks for height
    if height < 9 and height > 0:
        break

for i in range (0, height, 1): #itinerates rows
    for k in range (height-1, -1, -1):  #1st piramid colums
        if i < k:
            print(" ", end="")
        else:
            print("#", end="")
    print("  ", end="") #second piramid colums
    for j in range (-1, height - 1, 1):
        if i > j:
            print("#", end="")
        else:
            print(" ", end="")
    print() #change 